lib/shared/bucketing-assembly-script/__tests__/eventQueue/eventQueueManager.test.ts

```diff
@@ -44,7 +46,12 @@ const initSDK = (envKey: string, eventOptions: unknown = {}) => {
 }
 
 describe('EventQueueManager Tests', () => {
-    afterEach(() => clearPlatformData())
+    afterEach(() => {
+        clearPlatformData()
+        if (currentEnvKey) {
+            cleanupEventQueue(currentEnvKey)
+        }
```

Comment:
Should `currentEnvKey` also be cleared here?

Reply:
it can be yea.